### ISSUE TITLE
Handle automated-attendant DTMF phone extensions (iCloud pause/wait and RFC3966 phone extensions)

### ISSUE THREAD

**Issue by bertrand-lupart** [64characters/Telephone#67]
RFC3966 describes phone extensions to be used with tel: URI schemes that don't seem to work (1.0.4 tested)

``` html
<a href="tel:+1-555-438-3732;ext=1234">1-555-IETF-RFC</a>
```

iOS unofficial address book way to handle such extensions (+1-555-438-3732,1234) doesn't seem work either.

**Comment by eofster** [comment]
> if typed into the dial field, +1-555-438-3732 is called but 1234 is never sent to the remote phone system (and there's no way to manually enter it)

Do you mean that when you try to send DTMF manually, it is not received?

**Comment by bertrand-lupart** [comment]
> Do you mean that when you try to send DTMF manually, it is not received?

I'm not sure. I guess at this time i was expecting some graphical keyboard to be available, and didn't noticed it was possible to enter the digit with the keyboard.

I guess this was a pointless side remark.

**Comment by eofster** [comment]
Yeah, the current way of sending DTMF is impossible to find. The future UI updates should make this feature more obvious. Do I understand correctly, that the original problem was that this feature was nowhere to be seen, and not that you tried to send DTMF, but it didn’t work?

**Comment by bertrand-lupart** [comment]
Hello,

The original problem was to handle automated-attendant :
https://en.wikipedia.org/wiki/Automated_attendant

iOS allows to register numbers with soft pause and hard pause, using comma or semicolon :
https://support.apple.com/en-us/HT202176

+1-555-438-3732,1234
- compose +1-555-438-3732
- wait for the callee to answer
- automatically send '1234' DTMF

+1-555-438-3732;1234

- compose +1-555-438-3732
- wait for the callee to answer
- "dial 1234" button appears on iPhone screen for the caller to send '1234' DTMF

You end up with this kind of numbers in iCloud contacts, and Telephone don't know how to handle them : it compose all the numbers in a row without pausing, and the call fails.

Apple way doesn't seem to be the standard way described in RFC3966 :
`<a href="tel:+1-555-438-3732;ext=1234">1-555-IETF-RFC</a>
`
This would allow to compose +1-555-438-3732 by clicking on a link, wait for the call to answer and automatically send 1234 DTMF.

I can provide a french automated-attendant number for testing purpose.

Thank you,
